### DISCUSSION TITLE
FFMPEG wont start when opening video cameras

### DISCUSSION THREAD

**Discussion by OneJaeAtATime** [homebridge-simplisafe3/homebridge-simplisafe3#425] [Q&A]
### Description

I'm getting an error when starting the camera on Home app on iOS.

### Expected behavior

Expecting to see and hear the camera in the Home App.

### Steps to reproduce

1. Open Home App
2. Open Camera in Home app

### Homebridge logs

```bash
[SimpliSafe] Unable to spawn ffmpeg process at undefined with error: TypeError: The "file" argument must be of type string. Received undefined
    at validateString (node:internal/validators:162:11)
    at normalizeSpawnArguments (node:child_process:548:3)
    at spawn (node:child_process:754:13)
    at StreamingDelegate.handleStreamRequest (/var/lib/homebridge/node_modules/homebridge-simplisafe3/lib/streamingDelegate.js:372:48) {
  code: 'ERR_INVALID_ARG_TYPE'
```

### Homebridge config.json contents

```json
{
            "name": "SimpliSafe",
            "cameras": true,
            "sensorRefresh": 15,
            "persistAccessories": true,
            "cameraOptions": {
                "enableHwaccelRpi": true
            },
            "platform": "homebridge-simplisafe3.SimpliSafe 3"
        },
```

### Plugin version

v1.10.13

### Homebridge version

1.7.0

### Node.js version

v20.10.0

### Host OS

Raspbian

**Comment by shamoon** [comment]
Oh I didn’t even notice this. Your ffmpeg didn’t install correctly or isn't supported. That is entirely handled by a tool we use, ffmpeg-for-homebridge, not this plugin
